examples/stake-tokens/api/annual-provisions.ts

```diff
@@ -0,0 +1,17 @@
+type AnnualProvisions = {
+  annual_provisions: string;
+};
+
+export const handleError = (resp: Response) => {
+  if (!resp.ok) throw Error(resp.statusText);
+  return resp;
+};
+
+export const getAnnualProvisions = async (
+  chainName: string
+): Promise<AnnualProvisions> => {
+  const url = `https://lcd-${chainName}.keplr.app/cosmos/mint/v1beta1/annual_provisions`;
```

Comment:
Do we need to use this? This seems a bit of an anti-pattern since we're already using RPC clients that have well defined URLs and a system for making sure those endpoints work.

Here we're not making a request outside of the RPC clients. Can we instead query this via the RPC client? 

There are good chances for some chain-registry chains, that `lcd-${chainName}.keplr.app` doesn't exist. We should rely on the telescope-based generated RPC code